lib/matplotlib/axes/_base.py

```diff
@@ -3145,11 +3143,9 @@ def set_xscale(self, value, **kwargs):
         """
         Set the x-axis scale.
 
-        .. ACCEPTS: [ 'linear' | 'log' | 'symlog' | 'logit' | ... ]
-
         Parameters
         ----------
-        value : {"linear", "log", "symlog", "logit"}
+        value : {"linear", "log", "symlog", "logit", ...}
```

Comment:
I know there is an ellipsis above, but what does it mean?

Reply:
It's all the scales registered in the _scale_mapping dict.